#颜色识别参数 小广角鱼眼
color_param:
  green: [50,104,169,71,200,200]
  blue: [102,176,173,110,255,255]
  red: [156,98,194,255,255,255]
  yellow: [25,130,200,30,255,255]
  # orange: [7,100,189,11,255,255]
#底板识别参数
palte_color_param:
  green: [50,45,45,80,255,255]
  blue: [104,90,90,110,255,255]
  red: [156,60,60,255,255,255]
  yellow: [25,45,45,35,255,255]
  # orange: [7,100,189,11,255,255]
template: "template"    #定位板模板图片名称
bin_param:
  Binary: [120,300]
matchShapes_value: 0.1

#定位参数
palte_det_time: 8.0     #底板检测超时时间
times_of_palte_det: 3   #底板检测重复次数
palte_det_err: 0.03     #底板检测误差

color_det_time: 2.0     #颜色探测超时时间
color_loc_time: 4.0     #颜色定位超时时间
color_loc_times: 5      #重复定位次数

err: 5                  #像素重复精度
err_a: 3.0              #角度重复精度
err_times: 3            #长度错误次数

#机械臂
g_open: 0               #机械臂打开时的舵机度数
arm_query_time: 0.2     #机械臂查询关节位置周期
arm_wait_para: 2.0      #机械臂运动超时时间系数
arm_arrive_err: 10      #机械臂运动到达关节误差值
grip_query_time: 0.2    #机械臂夹具查询关节位置周期
grip_wait_time: 3.0     #机械臂夹具运动超时时间
gripper_arrive_err: 10  #机械臂夹具运动到达关节误差值


# 备份
# 新鱼眼
# color_param:
#   green: [50,94,169,71,200,200]
#   blue: [105,158,145,110,255,255]
#   red: [178,101,152,255,255,255]
#   yellow: [25,125,194,30,255,255]
#   # orange: [7,100,189,11,255,255]
# #底板识别参数
# palte_color_param:
#   green: [50,45,45,71,255,255]
#   blue: [105,90,90,110,255,255]
#   red: [178,60,60,255,255,255]
#   yellow: [25,45,45,30,255,255]
#   # orange: [7,100,189,11,255,255]